packages/pwa-kit-react-sdk/src/ssr/server/opentelemetry-server.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {NodeTracerProvider} from '@opentelemetry/sdk-trace-node'
+import {SimpleSpanProcessor} from '@opentelemetry/sdk-trace-base'
+import {B3Propagator} from '@opentelemetry/propagator-b3'
+import {Resource} from '@opentelemetry/resources'
+import {SemanticResourceAttributes} from '@opentelemetry/semantic-conventions'
+import {propagation} from '@opentelemetry/api'
+
+const SERVICE_NAME = 'pwa-kit-react-sdk'
+
+let provider = null
+
+/**
+ * Initialize OpenTelemetry tracing for server-side rendering
+ * @returns {NodeTracerProvider|null} The initialized provider or null if initialization failed
+ */
+export const initializeServerTracing = () => {
+    try {
+        // Initialize the tracer provider
+        provider = new NodeTracerProvider({
+            resource: new Resource({
+                [SemanticResourceAttributes.SERVICE_NAME]: SERVICE_NAME
```

Comment:
Did you look into the deprecation warnings?

It looks like SemanticResourceAttributes is deprecated an we should instead `SEMRESATTRS_SERVICE_NAME`